### PR TITLE
Chapter 8

### PR DESCRIPTION
**Pull request by cameronmccord2**
I finished chapter 8 and its tutorials. All tests are passing.

### CODE REVIEW

**Comment by jpotts18** [comment]
Try to have a couple more commit messages.

File: sample_app/app/controllers/users_controller.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def show
   def create
   	@user = User.new(user_params)
   	if @user.save
+      log_in @user
```
Comment (jpotts18) [inline sample_app/app/controllers/users_controller.rb:11]: Is this supposed to be here?

File: sample_app/test/integration/users_login_test.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  	post login_path, session: {email: @user.email, password: "password"}
+  	assert_redirected_to @user
+  	follow_redirect!
+  	assert is_logged_in?
+  	assert_template 'users/show'
+  	assert_select "a#logout", 1
+  	assert_select "a#profile", 1
+  	assert_select "a#login", 0
+  	assert_select "a[href=?]", login_path, count: 0
+    assert_select "a[href=?]", logout_path
```
Comment (jpotts18) [inline sample_app/test/integration/users_login_test.rb:36]: indentation?

File: sample_app/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[*.rb]
+indent_style = tab
+indent_size = 2
+
+[*.erb]
+indent_style = tab
+indent_size = 2
```
Comment (jpotts18) [inline sample_app/.editorconfig:7]: I don't think this file should be in the repository

**Comment by jpotts18** [comment]
Please read over those comments and make changes if you would like. Thank for the PR!

File: sample_app/app/controllers/users_controller.rb (hunk 1 above)
Comment (cameronmccord2) [inline sample_app/app/controllers/users_controller.rb:11]: Yes. When a new user gets created in the system we automatically log them in. I think in a later chapter we get rid of that in favor of waiting on email verification first.

**Pull request merged**